scripts/datastore_upgrade.py

```diff
@@ -409,6 +411,32 @@ def all_services_started(status_dict):
       return False
   return True
 
+def ensure_cassandra_nodes_match_replication(keyname):
+  # Get the database master IP.
+  with open(MASTERS_FILE_LOC) as masters_file:
+    master_ip = masters_file.read().strip()
+
+  command = cassandra_interface.NODE_TOOL + " " + 'status'
+  key_file = '{}/{}.key'.format(utils.KEY_DIRECTORY, keyname)
+  ssh_cmd = ['ssh', '-i', key_file, master_ip, command]
+  cmd_output = subprocess.Popen(ssh_cmd, stdout=subprocess.PIPE)
```

Comment:
Just for future reference, you can use the `subprocess.check_output` convenience function when you need the output.